.circleci/config.yml

```diff
@@ -583,12 +602,14 @@ workflows:
             . venv/bin/activate
             export PATH=${PATH}:${PWD}/tests/integration/bin:/opt/vertica/bin
             export DB_FACTS_PATH=${PWD}/tests/integration/circleci-dbfacts.yml
+            echo $SCRATCH_GCS_URL  # TODO
+            echo $SCRATCH_S3_URL   # TODO
             export RECORDS_MOVER_SESSION_TYPE=env
             mkdir -p test-reports/itest
             cd tests/integration
             python3 -m records.multi_db.test_records_table2table
-          requires:
-            - redshift-s3-itest
+#          requires:  # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker